templates/default.conf

```diff
@@ -13,6 +13,11 @@ map $request_uri $new_uri {
   include /etc/nginx/conf.d/redirects[.]map;
 }
 
+# We should use $host$request_uri to support redirect when nginx is serving multiple domains.
+map $host$request_uri $new_host_and_uri {
+  include /etc/nginx/conf.d/redirects[.]map;
+}
+
```

Comment:
we need sommething here:

1. Add a CHANGELOG.md
2. Add an example to explain how to write redirects

Reply:
Done